### PR TITLE
Banning users

### PR DESCRIPTION
**Pull request by MCSH**
It might be a good idea to ignore prs from users wjo are banned.

Ideas for further improvement:
* add a poll method to ban a user (using issues preferably)
* deny prs from banned users

### CODE REVIEW

**Comment by romgrk** [comment]
Ban Zidail :D (The guy from https://github.com/chaosbot/chaos/pull/48)

**Comment by rudehn** [comment]
An alternative, less harsh, approach would be to at least have chaosbot comment on PRs from people with previous PRs deemed really bad by the community.

Could be as simple as "Hey this guy might be malicious/spam".

This gives a heads up for everybody

**Comment by MCSH** [comment]
@rudehn that sounds interesting. I'll work on it later.

**Comment by MINIMAN10000** [comment]
I prefer rudehn's idea for chaosbot leaving a comment on PRs

I think part of the fun in chaosbot is everyone has a vote. Made me happy when I read that democracy vote got accepted. ( Removal of following bonus )

**Comment by MCSH** [comment]
@MINIMAN10000 interestingly the pr got reverted due to server error.

f2cd29ea15de881ab3377cad5519e0c2dcfe00ac

Coincidence?

File: github_api/prs.py [hunk 1]
```diff
@@ -100,6 +100,7 @@ def get_ready_prs(api, urn, window):
     open_prs = get_open_prs(api, urn)
     for pr in open_prs:
         pr_num = pr["number"]
+        user = pr["user"]["id"]
```
Comment (amoffat) [inline github_api/prs.py:103]: use "login" instead of "id"... it's pretty tricky to find your id

**Review by amoffat — CHANGES_REQUESTED** [review]

File: github_api/prs.py (hunk 1 above)
Comment (MCSH) [reply github_api/prs.py:103]: You can change login

Comment (amoffat) [reply github_api/prs.py:103]: oh wow, did not know

**Comment by amoffat** [comment]
If anyone is wondering why this got rejected, with #42 put in place, and with more yays than nays, the answer is [here](https://github.com/chaosbot/chaos/blob/master/patch.py#L36).  User vote weights are cached for a day.

**Pull request closed without merging**